spec/models/comment_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'Validations For the Comment model' do
+    before(:each) do
+      @comment = Comment.new(Text: 'Future programmer', author_id: 1)
+    end
+    before { @comment }
+
+    it 'for author_id not present' do
+      @comment.author_id = nil
+      expect(@comment).to_not be_valid
+    end
+    it 'for title when present' do
+      @comment.Text = nil
+      expect(@comment).to_not be_valid
+    end
+
+    it 'for author_id to be present' do
+      @comment.author_id = false
+      expect(@comment).to_not be_valid
+    end
+
+    it 'author_id is integer' do
+      @comment.author_id = 'joseph'
+      expect(@comment).to_not be_valid
+    end
+  end
+end
```

Comment:
- [ ] Please make sure to test all methods implemented in the `Comment` model, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods) requirement.